tutorial/advanced/t15_finetuning.ipynb

```diff
@@ -617,15 +608,17 @@
        "<All keys matched successfully>"
       ]
      },
-     "execution_count": 15,
+     "execution_count": 16,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
+    "weights_path=os.path.join(model_dir, 'lenet_torch.pt')\n",
+    "\n",
     "model_torch_pretrained = lenet_torch(input_shape=torch_input_shape, classes=base_num_classes)\n",
     "\n",
-    "model_torch_pretrained.load_state_dict(load(torch_pretrained_model_path))"
+    "model_torch_pretrained.load_state_dict(load(weights_path, map_location='cpu' if cuda.device_count() == 0 else None))"
```

Comment:
suppose I train on 2 gpu machine and then load weights on 1 gpu machine. would this still work? seems like this only catches the case going from multi-gpu down to cpu